src/tests/functional/plugin/gpu/shared_tests_instances/single_layer_tests/tile.cpp

```diff
@@ -15,6 +15,7 @@ const std::vector<InferenceEngine::Precision> netPrecisions = {
 };
 
 const std::vector<std::vector<int64_t>> repeats = {
```

Comment:
Are there any tests where the number of elements in "repeats" is more than shape of "data"?